docs/Getting-Started.md

```diff
@@ -1,13 +1,18 @@
 <h1 align="center">Fastify</h1>
 
 ## Getting Started
+Hello! Thank you for checking out Fastify!  
+This document aims to be a gentle introduction to the framework and its features. You will find some easy explanation, examples and link to other parts of the documentation if you want to get deepen in some argument.  
+Let's start!
 
+<a name="install"></a>
```

Comment:
=> `<a id="install"></a>`

See: https://jrfom.com/2015archive/2010/10/29/dont-name-anchor-elements/

Reply:
mmh, all our documentation uses `name`.
If you think we should change that I think we should do that in a separate pr.

Reply:
I've just been using the approach of "fix it as it comes in".